hs_core/migrations/0038_doi_url.py

```diff
@@ -0,0 +1,26 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+from hs_core.models import Identifier
+
+from django.db import migrations
+
+
+def remove_dx(apps, schema_editor):
+    for id in Identifier.objects.filter(url__startswith='http://dx.'):
+        id.url = id.url.replace('http://dx.', 'https://')
```

Comment:
Do we have any resources that start with `https://dx.`?

Reply:
no, every doi was hardocded to `http://dx.`.  https://github.com/hydroshare/hydroshare/commit/8f0a5f2815c47278044a2382053278bd49557407 shows that @katrinleinweber put in some work months ago to change new DOIs to `https://` which was merged to develop but never pushed out.  

I also ran a filter on `https://dx.` just to see if there were any matches and I did not find any.

Reply:
I used [`https?:\/\/(dx)?\.doi\.org`](https://regexper.com/#https%3F%3A\%2F\%2F(dx\.)%3Fdoi\.org\%2F) to catch both variants back then ;-)